cvat/apps/dataset_manager/tests/test_rest_api_formats.py

```diff
@@ -0,0 +1,1128 @@
+                            tmp_annotations[item][index_elem]["attributes"].append({
+                                "spec_id": spec_id,
+                                "value": value,
+                            })
+
+        response = self._put_api_v1_task_id_annotations(task["id"], tmp_annotations)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def _create_annotations_in_job(self, task, job_id,  name_ann, key_get_values):
+        tmp_annotations = copy.deepcopy(annotations[name_ann])
+
+        # change attributes in all annotations
+        for item in tmp_annotations:
+            if item in ["tags", "shapes", "tracks"]:
+                for index_elem, _ in enumerate(tmp_annotations[item]):
+                    tmp_annotations[item][index_elem]["label_id"] = task["labels"][0]["id"]
+
+                    for index_attribute, attribute in enumerate(task["labels"][0]["attributes"]):
+                        spec_id = task["labels"][0]["attributes"][index_attribute]["id"]
+
+                        if key_get_values == "random":
+                            if attribute["input_type"] == "number":
+                                start = int(attribute["values"][0])
+                                stop = int(attribute["values"][1]) + 1
+                                step = int(attribute["values"][2])
+                                value = str(random.randrange(start, stop, step))
+                            else:
+                                value = random.choice(task["labels"][0]["attributes"][index_attribute]["values"])
+                        elif key_get_values == "default":
+                            value = attribute["default_value"]
+
+                        if item == "tracks" and attribute["mutable"]:
+                            for index_shape, _ in enumerate(tmp_annotations[item][index_elem]["shapes"]):
+                                tmp_annotations[item][index_elem]["shapes"][index_shape]["attributes"].append({
+                                    "spec_id": spec_id,
+                                    "value": value,
+                                })
+                        else:
+                            tmp_annotations[item][index_elem]["attributes"].append({
+                                "spec_id": spec_id,
+                                "value": value,
+                            })
+
+        response = self._put_api_v1_job_id_annotations(job_id, tmp_annotations)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def _download_file(self, url, data, user, file_name):
+        for _ in range(5):
+            response = self._get_request_with_data(url, data, user)
+            if response.status_code == status.HTTP_200_OK:
+                content = BytesIO(b"".join(response.streaming_content))
+                with open(file_name, "wb") as f:
+                    f.write(content.getvalue())
+                break
+        return response
```

Comment:
I think, instead of a loop, we can do [this](https://github.com/openvinotoolkit/cvat/blob/f66212604473a5f65561e002091c1065b8de5656/cvat/apps/engine/tests/test_rest_api.py#L4625L4640)

Reply:
Done.